lib/ratelimitspec.go

```diff
@@ -0,0 +1,66 @@
+package lib
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// RateLimitSpec defines how rate limit is expressed
+type RateLimitSpec struct {
+	// burstSize defines the size of bursts for the period
+	BurstSize int
+	// Period of ticker
+	Period time.Duration
+}
```

Comment:
If we do want to support multiple fields of config, I think it might be better to accept fields instead of creating a micro-format for this. But we can figure out the UX once we settle on the rate limiter implementation.

Reply:
Burstsize, might be configurable, but I hard-coded at 2 for now. For pure cache, I don't think it deserves more configuration as I don't see how an operator might need to tune it for now (might be useful in the future - who knows)

Regarding time, see https://github.com/hashicorp/consul/issues/7863#issuecomment-631455816 => I think expressing it as "query per something" is nicer to end-user (aka: 1/s or 1/10s), but I don't care, I have been told to express it as query per something in the issue, so I did.

Reply:
I use the same config type as RPCLImit now